mozdownload/__init__.py

```diff
@@ -2,4 +2,4 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from scraper import *
+from scraper import * #noqa
```

Comment:
I assume it's because of the `import *`? Maybe we should simply state every supported scraper class.